code/modules/mob/living/simple_animal/revenant/revenant.dm

```diff
@@ -216,7 +216,7 @@
 	..()
 
 /datum/objective/revenant/check_completion()
-	if(!istype(owner.current, /mob/living/simple_animal/revenant) || !owner.current)
+	if(owner && !istype(owner.current, /mob/living/simple_animal/revenant) || !owner || !owner && !owner.current)
```

Comment:
This doesn't make any sense.

The first part does; Can't istype owner.current if owner doesn't exist.
But... then it goes to `!owner || !owner && !owner.current`.